.gitlab-ci-check-docker-acceptance.yml

```diff
@@ -101,6 +106,8 @@ test:acceptance_tests:
     paths:
       - tests/acceptance.*
       - tests/coverage-acceptance.txt
+    reports:
+      junit: tests/results.xml
```

Comment:
will this be generated by the common tester image?

Reply:
Not the image itself, but the [wrapper script](https://github.com/mendersoftware/deviceauth/blob/master/tests/run.sh#L21) in each repository. I *think* CI won't fail the job is the file is missing, it will just not parse it.

You can see it working [here](https://gitlab.com/Northern.tech/Mender/deviceauth/-/pipelines/292362776/test_report).

Reply:
thanks for the pointers!